Mapsui/Navigator.cs

```diff
@@ -647,14 +662,33 @@ private void AddToInitialization(Action action)
         });
     }
 
+    private bool ShouldInitialize() => !IsInitialized && CanInitialize();
+
+    private bool CanInitialize() => Viewport.HasSize() && PanBounds is not null; // Should we check on ZoomBounds as well?
```

Comment:
Not sure what to do with the ZoomBounds. They can be relevant for limiting, but they do not have to be specified.